src/pubsub/peer-streams.js

```diff
@@ -9,6 +9,7 @@ const log = Object.assign(debug('libp2p-pubsub:peer-streams'), {
 /** @type Events */
 const EventEmitter = require('events')
 
+// @ts-ignore TODO: https://github.com/alanshaw/it-length-prefixed/pull/15
```

Comment:
This has been merged, can be removed?

Reply:
I am waiting on releasing this: https://github.com/alanshaw/it-length-prefixed/pull/16
Otherwise tests fail